middlewares/auth.js

```diff
@@ -0,0 +1,16 @@
+const passport = require("passport");
+
+module.exports = (req, res, next) => {
+  passport.authenticate("jwt", { session: false }, (err, user) => {
+    if (!user || err) {
```

Comment:
Warto upewnić się, że użytkownik posiada ważny token uwierzytelniający przed przyznaniem dostępu do zasobów aplikacji. Jeśli token jest nieprawidłowy lub nieobecny, użytkownik powinien otrzymać odpowiedź 401 (Not Authorized).

Reply:
wprowadziłem poprawki choć nie jestem pewien czy to dokładnie chodziło

Reply:
wystarczy że sprawdzisz czy user.token zawiera dane.